variables.tf

```diff
@@ -3,7 +3,7 @@
 ##############################################################################
 
 variable "resource_group_id" {
-  description = "id of resource group to create VPC"
+  description = "id of resource group to create VPC and volumes"
```

Comment:
Also, since you are still supporting optionally setting the resource group in the `block_storage_volumes` variable, you should maybe mention that in this description. Propose:

> ID of resource group to create VSI and block storage volumes. If you wish to create the block storage volumes in a different resource group, you can optionally set that directly in the  'block_storage_volumes' variable.